compiler/rustc_codegen_rmc/src/context/goto_ctx.rs

```diff
@@ -160,15 +162,18 @@ impl<'tcx> GotocCtx<'tcx> {
     /// Returns: a struct-tag referencing the inserted struct.
     pub fn ensure_struct<F: FnOnce(&mut GotocCtx<'tcx>, &str) -> Vec<DatatypeComponent>>(
         &mut self,
-        struct_name: &str,
-        pretty_name: Option<String>,
+        struct_name: &str, //DSN
```

Comment:
?

Reply:
That's my marker that I haven't completed porting this function yet.